canal/interconnect.py

```diff
@@ -282,7 +282,7 @@ def __ground_ports(self):
                     # no connection to that sb port, ground it
                     sb_name = create_name(str(sb))
                     sb_port = self.tile_circuits[coord].ports[sb_name]
-                    self.wire(ground, sb_port[0])
+                    self.wire(ground, sb_port)
```

Comment:
@Kuree  do we need to add similar checeks here and in line 265?

Reply:
Yes you're right. I didn't had that before. I will fix it.

Reply:
No actually this would never happen. This is boundary switch boxes where there is no connection coming in. As a result, the size will always be 1 (0 actually, but will be upcast to 1 in the old mux implementation).

Just need to add the upcast for the new mux.